src/containers/index.js

```diff
@@ -1,7 +1,13 @@
 import LandingPage from './LandingPage';
 import LoginPage from './LoginPage';
+import SignupPage from './SignupPage';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/n4alphapesto/hirebook-frontend/pullRequest?prid=7756469)
```suggestion
import SignupPage from "./SignupPage";
```